scripts/cybershake/auto_submit.py

```diff
@@ -301,11 +331,32 @@ def main():
         task_state = db_task_status[2]
 
         # skip im calcs if no_im == true
-        if args.no_im and proc_type == 6:
+        if args.no_im and proc_type == const.ProcessType.IM_calculation.value:
             task_num = task_num + 1
             continue
-        if args.no_merge_ts and proc_type == 2:
-            update_mgmt_db.update_db(db, "merge_ts", "completed", run_name=run_name)
+        if proc_type == const.ProcessType.merge_ts.value:
+            if args.no_merge_ts:
+                update_mgmt_db.update_db(
+                    db, "merge_ts", const.State.completed.value[1], run_name=run_name
+                )
+                task_num = task_num + 1
+                continue
+            elif slurm_query_status.is_task_complete(
+                [
+                    const.ProcessType.clean_up.value,
+                    run_name,
+                    const.State.completed.value[1],
+                ],
+                slurm_query_status.get_db_tasks_to_be_run(db),
```

Comment:
Rather than querying here, leave the second argument as a optional parameter and run the command as the default if it doesn't exist. 

I think it's better separation of logic.